spec/tasks/gen/action_spec.cr

```diff
@@ -3,31 +3,35 @@ require "../../spec_helper"
 include CleanupHelper
 
 describe Gen::Action do
-  it "generates a basic action" do
+  it "generates a basic browser action" do
     with_cleanup do
       io = IO::Memory.new
       valid_action_name = "Users::Index"
       ARGV.push(valid_action_name)
 
-      Gen::Action.new.call(io)
+      Gen::Action::Browser.new.call(io)
+
+      index_action = File.read("./src/actions/users/index.cr")
+      index_action.should contain(valid_action_name)
+      index_action.should contain("< BrowserAction")
 
-      File.read("./src/actions/users/index.cr")
-          .should contain(valid_action_name)
       io.to_s.should contain(valid_action_name)
       io.to_s.should contain("/src/actions/users")
     end
   end
 
-  it "generates a nested action" do
+  it "generates a nested api action" do
```

Comment:
I think it would be helpful to have a test for generating nested browser actions as well